app/layout.tsx

```diff
@@ -10,8 +10,8 @@ import { Toaster } from "react-hot-toast";
 const inter = Inter({ subsets: ["latin"] });
 
 export const metadata = {
```

Comment:
It seems like the constant with data should be moved to /lib/data.ts. This aligns with best practices for organizing code and separating concerns.